pkg/bundle/files.go

```diff
@@ -130,7 +131,8 @@ func ReadImageSet(a archive.Archiver, from string) (map[string]string, error) {
 			if extension == a.String() {
 				logrus.Debugf("Found archive %s", path)
 				return a.Walk(path, func(f archiver.File) error {
-					filesinArchive[f.Name()] = path
+					name := filepath.Clean(f.Header.(*tar.Header).Name)
```

Comment:
Recomending a check here, since if for some reason `f.Header` does not hold type `tar.Header` this will panic.
```suggestion
                                         header, ok := f.Header.(*tar.Header)
                                         if !ok {
                                         return fmt.Errorf("invalid tar header for file %s", f.Name())
                                         }
					name := filepath.Clean(header.Name)
```